test/image-blueprints-bootc/layer1-base/group2/rhel100-bootc-brew-ec-with-optional.containerfile

```diff
@@ -0,0 +1,53 @@
+# {{- if env.Getenv "BREW_EC_RELEASE_VERSION" "" -}}
+# Note: This comment makes templating add a new line before the code
+FROM localhost/rhel100-test-agent:latest
+
+# Build arguments
+ARG USHIFT_RPM_REPO_NAME=microshift-brew
+ARG USHIFT_RPM_REPO_PATH=/tmp/$USHIFT_RPM_REPO_NAME
+
+# Copy the MicroShift repository contents
+COPY ./rpm-repos/$USHIFT_RPM_REPO_NAME $USHIFT_RPM_REPO_PATH
+
+# Copy repository configuration
+COPY ./bootc-images/$USHIFT_RPM_REPO_NAME.repo ./bootc-images/microshift-fast-datapath-rhel10.repo ./bootc-images/microshift-rhocp-y.repo \
```

Comment:
We are installing RHEL 9 RPMs. As such, we need to have only RHEL 9 fast-datapath repo.
See the existing test/image-blueprints-bootc/layer2-presubmit/group1/rhel100-bootc-source.containerfile